service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/gcp/ainotebook/ControlledAiNotebookInstanceResource.java

```diff
@@ -136,7 +140,7 @@ public void addCreateSteps(
         gcpRetryRule);
     flight.addStep(
         new CreateAiNotebookInstanceStep(
-            this, petSaEmail, flightBeanBag.getCrlService(), flightBeanBag.getCliConfiguration()),
+            this, petSaEmail, userFacingWorkspaceId, flightBeanBag.getCrlService(), flightBeanBag.getCliConfiguration()),
```

Comment:
I *think* passing the value to this step is the right approach for a couple reasons:

- This value is mutable, so I don't think we should store it as a property of a resource like we do for workspace UUID. If we did, we would need to update all resources every time we change the UFID.
- As far as I know this is the only step that will need the user facing ID, so I don't think we should move this to a flight-level parameter or change the `addCreateSteps` signature that we use for all resource types.

I'm not huge on putting too much logic here though, open to other opinions on this part!